File: Big09/Big09/src/db연결/Db연결테스트.java
package db연결;

import java.sql.DriverManager;

public class Db연결테스트 {

	public static void main(String[] args) throws Exception {
		//외부 API 내 클래스 끌고올때는 '예외처리'필요
		//예외처리 적용사항: 외부파일, db연결, 네트워크 등의 외부사항
		//필요이유: 마이너에러 발생시 계속처리 진행 강제를 위해
		//트라이-캐치귀찮아, 에러나면 나오긴하지만, 메인 호출괄호 다음에 throws 지정 + 적용범위선택으로 처리더편함
		Class.forName("com.mysql.jdbc.Driver");	
		System.out.println("1.커넥팅성공");
		
		//디비버에서 디비명우측클릭, 에디터드라이브셋팅 후 하단처럼 정보찔러준다.
		String url = "jdbc:mysql://localhost:3306/phone";
		String user = "root";
		String password = "1234";
		
		//2 디비연결
		DriverManager.getConnection(url, user, password);
		System.out.println("2. db연결성공....");
	}

}

File: Big43/DB연결/db연결테스트.py
import pymysql as my

# 1. connector를 통해 db연결
# mysql과 연결
con = my.connect(host = "localhost", user = "root", password = "1234", db = "phone")
cursor = con.cursor() # 연결스트링을 만든다 # cursor 를 통해 sql문을 실행

# 2. sql문을 결정
sql = "insert into member2 values ('python', 'python', 'python', 'python')"
cursor.execute(sql)
print("sql문 실행 요청 완료!!!!!")

con.commit()
con.close()

# 3. sql문을 실행